third-party-drivers/phantomjs.rst

```diff
@@ -0,0 +1,85 @@
+PhantomJSDriver
```

Comment:
I don't think we should be maintaining the doc for third-party drivers ourselves. The list of third-party drivers should have external links to your doc instead